### DISCUSSION TITLE
Does MikroORM support fake deletions? SoftRemove is similar to TypeORM.

### DISCUSSION THREAD

**Discussion by Lydanne** [mikro-orm/mikro-orm#2421] [Q&A] [answered]
Does MikroORM support fake deletions? SoftRemove is similar to TypeORM.

**Comment by B4nan** [comment, marked as answer]
You can use onFlush event to alter pretty much anything, including the change from delete to update. 

https://mikro-orm.io/docs/lifecycle-hooks/#using-onflush-event
https://github.com/mikro-orm/mikro-orm/issues/1492#issuecomment-785394397

And you can filters for ignoring such soft deleted entities:

https://mikro-orm.io/docs/filters/
https://github.com/mikro-orm/mikro-orm/issues/385
